packages/react-instantsearch-hooks-dom/src/ui/Hits.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { Hit } from 'instantsearch.js';
+import type { JSXElementConstructor } from 'react';
+
+export type HitsProps<THit> = React.ComponentProps<'div'> & {
+  hits: THit[];
+  hitComponent?: JSXElementConstructor<{ hit: THit }>;
```

Comment:
Why import this type instead of referencing it from `React` like you did with `ComponentProps`?